### PR TITLE
FEED-142: Moving away from deprecated Django-image

### PR DESCRIPTION
**Pull request by hbokh**
Docker image "django:latest" is deprecated since Dec. 2016.
This is a new (and slightly smaller) image.
So far it shows the same behaviour as the Django-image.

### CODE REVIEW

**Comment by hbokh** [comment]
`psycopg2` can de dropped since we do not use the PostgreSQL database.

**Comment by hbokh** [comment]
**UPDATE**

- Fixed OAuth2-errors by upgrading to `django-rest-framework-social-oauth2==1.0.8`.
- File `base.py`:
  in `DEFAULT_AUTHENTICATION_CLASSES` changed `oauth2_provider.ext.rest...` to `oauth2_provider.contrib.rest...`.
- File `Dockerfile`:
   removed `psycopg2`.
- Removed duplicate `python-social-auth==0.3.4` in `backend/requirements.txt`.

*Works on my machine!* 🐳

File: backend/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,19 @@
...
+    mysql-client \
+    libmysqlclient-dev && \
+    apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+## We COULD install Django from here, but it will be overriden
+## by the requirements-file ...
+# ENV DJANGO_VERSION 1.11.3
+# RUN pip install mysqlclient django=="$DJANGO_VERSION"
+
+RUN pip install mysqlclient
```
Comment (ferdynice) [inline backend/Dockerfile]: I don't think we need this at all. If Django does not already this dependency, using a mysql client can be done from the db container.

Can you test if the stack works without it?

**Comment by ghost** [comment]
Works on my machine too!

**Review by ghost — APPROVED** [review]

**Pull request merged**